opds2_import.py

```diff
@@ -179,6 +178,21 @@ def __init__(
         self._parser = parser
         self._logger = logging.getLogger(__name__)
 
+    def _is_identifier_allowed(self, identifier: Identifier) -> bool:
+        """Check the identifier and return a boolean value indicating whether CM can import it.
+
+        NOTE: Currently, this method hard codes allowed identifier types.
+        The next PR will add an additional configuration setting allowing to override this behaviour
+        and configure allowed identifier types in the CM Admin UI.
+
+        :param identifier: Identifier object
+        :type identifier: Identifier
+
+        :return: Boolean value indicating whether CM can import the identifier
+        :rtype: bool
```

Comment:
Do we need the param/return type hints in the docstring, given the inline function/method type hint annotations? It means having to maintain those hints in two places (though they are at least close to each other).

This is minor and mostly expressing my own preference now that we're on Python 3.6+ and have at least PEP 484  hints.